source/en/learn/generalized-plasma.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Generalized Plasma
+lead: Why is EVM-on-Plasma hard?
+date: 2019-1-13 4:47:02
+categories:
+tags:
+links:
+  before:
+    Introduction: /en/learn
+    The Plasma Framework: /en/learn/framework.html
+    Plasma MVP: /en/learn/mvp.html
+    Plasma Cash: /en/learn/cash.html
+    Plasma Debit: /en/learn/debit.html
+  after:
+    Comparison: /en/learn/compare.html
+---
+
+## Generalized Plasma
+
+It is difficult to create a Plasma chain that operates a lot like Ethereum does today and is capable of running more general smart contracts. This page seeks to address the question, ["Why is EVM-on-Plasma hard?,"](https://medium.com/@kelvinfichter/why-is-evm-on-plasma-hard-bf2d99c48df7) with an introduction to possible solutions, such as the “Plasma VM” design introduced by researcher Kelvin Fichter and the "Plasma Leap" design being developed by LeapDAO.
```

Comment:
Would recommend that every sentence be separated into an individual line to make diffing easier when we modify content.

Reply:
Sure thing~definitely want ease-of-edits. Will make that tweak now 👍